docs/site/Creating-CRUD-REST-apis.md

```diff
@@ -158,7 +158,7 @@ module.exports = <ModelCrudRestApiConfig>{
 Under the hood, a default CRUD controller and repository are created using
 [`defineCrudControllerClass`](https://loopback.io/doc/en/lb4/apidocs.rest-crud.definecrudrestcontroller.html)
 and
-[`defineCrudRepositoryClass`](https://loopback.io/doc/en/lb4/apidocs.rest-crud.definecrudrepositoryclass.html),
+[`defineCrudRepositoryClass`](https://loopback.io/doc/en/lb4/apidocs.repository.defaultcrudrepository.html),
```

Comment:
Not sure if this is the right one. But the original link no longer exist.

Reply:
Here's the link:
```suggestion
[`defineCrudRepositoryClass`](https://loopback.io/doc/en/lb4/apidocs.repository.definecrudrepositoryclass.html),
```